### PR TITLE
Chore(charts): Unit tests for wardend. Updated pipelines, docs

### PR DESCRIPTION
**Pull request by mikolajsobolewski**

### CODE REVIEW

File: charts/wardend/templates/service.yaml [hunk 1]
```diff
@@ -89,9 +89,8 @@ spec:
     statefulset.kubernetes.io/pod-name: {{ include "node.name" $ }}-{{ $i }}
 ---
 {{- end }}
-{{- end }}
```
Comment (jlehtimaki) [inline charts/wardend/templates/service.yaml:92]: Why are these removed?

File: charts/wardend/tests/unit/ingress_test.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+suite: Ingress Template Tests
+templates:
+  - ingress.yaml  # Укажите путь к вашему шаблону, если он в другом месте
```
Comment (jlehtimaki) [inline charts/wardend/tests/unit/ingress_test.yaml:3]: Remove the russian texts

**Review by jlehtimaki — CHANGES_REQUESTED** [review]
Is there a way to add additional information to the chart README.
Now it just removes everything that has been written there automatically, which is not nice.

**Comment by mikolajsobolewski** [comment]
> Is there a way to add additional information to the chart README. Now it just removes everything that has been written there automatically, which is not nice.

Yes of course, you can create fully customizable file via README.md.gotmpl or add custom comments. I created #600 into infra for this task.

File: charts/wardend/templates/service.yaml (hunk 1 above)
Comment (mikolajsobolewski) [reply charts/wardend/templates/service.yaml:92]: i simplified `service.yaml` and kept `{{- if .Values.node.enabled }}` into only one place.

**Comment by jlehtimaki** [comment]
> > Is there a way to add additional information to the chart README. Now it just removes everything that has been written there automatically, which is not nice.
> 
> Yes of course, you can create fully customizable file via README.md.gotmpl or add custom comments. I created #600 into infra for this task.

But it seems, that it's overwriting everything that's now in some of the READMEs, is that by design now?

**Review by jlehtimaki — APPROVED** [review]
